doc/classes/Node.xml

```diff
@@ -1161,10 +1165,10 @@
 			Notification received when the [TextServer] is changed.
 		</constant>
 		<constant name="PROCESS_MODE_INHERIT" value="0" enum="ProcessMode">
-			Inherits [member process_mode] from the node's parent. For the root node, it is equivalent to [constant PROCESS_MODE_PAUSABLE]. This is the default for any newly created node.
+			Inherits [member process_mode] from the node's parent. This is the default for any newly created node.
 		</constant>
 		<constant name="PROCESS_MODE_PAUSABLE" value="1" enum="ProcessMode">
-			Stops processing when [member SceneTree.paused] is [code]true[/code]. This is the inverse of [constant PROCESS_MODE_WHEN_PAUSED].
+			Stops processing when [member SceneTree.paused] is [code]true[/code]. This is the inverse of [constant PROCESS_MODE_WHEN_PAUSED], and the default for the root node.
```

Comment:
Out of scope of the PR, but sure! You did this for consistency with the other enum, that's okay. I mean, it is technically a lie, but that's how it behaves.

Reply:
Still better than "For the root node, it is equivalent to `PROCESS_MODE_PAUSABLE`." in `PROCESS_MODE_INHERIT` before.